crates/services/gas_price_service/src/ports.rs

```diff
@@ -26,3 +26,26 @@ pub trait MetadataStorage: Send + Sync {
 pub trait GasPriceData: Send + Sync {
     fn latest_height(&self) -> Option<BlockHeight>;
 }
+
+pub struct GasPriceServiceConfig {
+    pub min_gas_price: u64,
+    pub starting_gas_price: u64,
+    pub gas_price_change_percent: u64,
+    pub gas_price_threshold_percent: u64,
+}
+
+impl GasPriceServiceConfig {
```

Comment:
Is this going to be used in ports? 
If not, maybe it's best to move it either in a separate `config.rs` file or directly in `lib.rs`

Reply:
(comment refers both to the struct definition and implementation)

Reply:
well we define the implementation of the port (actually just the conversion from `Config` to `GasPriceServiceConfig`) in `crates/fuel-core/src/service/adapters/gas_price_adapters.rs` so to me it made sense to keep it all in one place :)